service/rtc/rate.go

```diff
@@ -43,11 +41,14 @@ func (m *RateMonitor) PushSample(size int) {
 	m.mut.Lock()
 	defer m.mut.Unlock()
 
-	if !m.filled && m.getSamplesDuration() < m.samplingSize {
+	// Filling up to double the sampling size to make sure we have enough samples
+	// to calculate the desired duration since at the beginning it's likely we get
+	// a burst of packets.
+	if !m.filled && m.getSamplesDuration() < m.samplingSize*2 {
```

Comment:
I'm wondering if we should maybe give it some time to stabilize, and then start measuring?